app/controllers/observations_controller/index.rb

```diff
@@ -114,12 +114,22 @@ def render_observation_search_results(pattern)
       search.errors.each do |error|
         flash_error(error.to_s)
       end
-      render("index", location: observations_path)
+      if params[:needs_id]
+        redirect_to({ controller: "/observations/identify", action: :index,
```

Comment:
Probably outside the scope of this PR:
It would be nice to supply a path here instead of hash.
That would required that our `ApplicationController#redirect_to` work with a string, just like [ActionController#redirect_to](nimmo-observations-needing-identifications).
I dont think it would need many changes.